roles.js

```diff
@@ -100,98 +38,132 @@ exports.init = (c) => {
         "k6qw8z6g": guild.roles.cache.get("716015547984117872"),
         "wr": guild.roles.cache.get("716014433121337504")
     };
-    botDev = guild.members.cache.get("81612266826379264").user;
     if (Object.values(roles).includes(undefined)) {
         sendError(message, "Couldn't find all roles.");
         process.exit(1);
     }
-    botDev.createDM();
-    try {
-        data = JSON.parse(fs.readFileSync(srcDataFile));
-        leaderboards = data.leaderboards;
-        users = data.users;
-    } catch { 
-        console.log("Failed to load things; run !reload all I guess");
-        return;
+
+    // Setup tables for keeping track of people's runs on sr.c
+    if (!sql.prepare("SELECT count(*) FROM sqlite_master WHERE type='table' AND name='runs';").get()['count(*)']) {
```

Comment:
Something like `src_runs` might be a better name so it's not confused with race results.

Reply:
I moved the roles stuff to its own sqlite file now